common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -277,14 +197,11 @@ const IIIFPresentationDisplay = ({
         previewThumbnails(
           iiifPresentationManifest,
           orderedStructuredImages(structuredImages(iiifPresentationManifest)),
-          4
+          5
```

Comment:
is this supposed to be `ideaNumber`?

Reply:
…I think you can ignore me – I was getting confused by the way GitHub collapsed the space from the function above.